src/main/java/com/sonymobile/tools/gerrit/gerritevents/dto/events/HashtagsChanged.java

```diff
@@ -0,0 +1,169 @@
+import static com.sonymobile.tools.gerrit.gerritevents.dto.GerritEventKeys.EDITOR;
+import static com.sonymobile.tools.gerrit.gerritevents.dto.GerritEventKeys.HASHTAGS;
+import static com.sonymobile.tools.gerrit.gerritevents.dto.GerritEventKeys.REMOVED;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import com.sonymobile.tools.gerrit.gerritevents.dto.GerritEventType;
+import com.sonymobile.tools.gerrit.gerritevents.dto.attr.Account;
+
+import net.sf.json.JSONArray;
+import net.sf.json.JSONObject;
+
+/**
+ * A DTO representation of the hashtags-changed Gerrit Event.
+ * @author David Pursehouse &lt;david.pursehouse@sonymobile.com&gt;
+ */
+public class HashtagsChanged extends ChangeBasedEvent {
+
+    /**
+     * The user who changed the hashtags.
+     */
+    private Account editor;
+
+    /**
+     * The hashtags that were added to the change.
+     */
+    private List<String> added;
+
+    /**
+     * The hashtags that were removed from the change.
+     */
+    private List<String> removed;
+
+    /**
+     * Hashtags currently on the change.
+     */
+    private List<String> hashtags;
+
+    @Override
+    public GerritEventType getEventType() {
+        return GerritEventType.HASHTAGS_CHANGED;
+    }
+
+    @Override
+    public boolean isScorable() {
+        return true;
+    }
+
+    /**
+     * Get the user who edited the hashtags.
+     * @return the user.
+     */
+    public Account getEditor() {
+        return editor;
+    }
+
+    /**
+     * Set the user who edited the hashtags.
+     * @param editor the user.
+     */
+    public void setEditor(Account editor) {
+        this.editor = editor;
+    }
+
+    /**
+     * Get the hashtags that were added to the change.
+     * @return the hashtags.
+     */
+    public List<String> getAdded() {
+        return added;
+    }
+
+    /**
+     * Set added hashtags.
+     * @param added the hashtags.
+     */
+    public void setAdded(List<String> added) {
+        this.added = added;
+    }
+
+    /**
+     * Get the hashtags that were removed from the change.
+     * @return the hashtags.
+     */
+    public List<String> getRemoved() {
+        return removed;
+    }
+
+    /**
+     * Set removed hashtags.
+     * @param removed the hashtags.
+     */
+    public void setRemoved(List<String> removed) {
+        this.removed = removed;
+    }
+
+    /**
+     * Get the hashtags on the change.
+     * @return the hashtags.
+     */
+    public List<String> getHashtags() {
+        return hashtags;
+    }
+    /**
+     * Set the hashtags on the change.
+     * @param hashtags the hashtags.
+     */
+    public void setHashtags(List<String> hashtags) {
+        this.hashtags = hashtags;
+    }
+
+    /**
+     * Convert a JSON array of Strings to a list of Strings.
+     * @param array the array to convert.
+     * @return list of Strings found in the Array.
+     */
+    private List<String> convertJSONArrayOfStringsToList(JSONArray array) {
```

Comment:
This could be moved to be together with the other json util methods in ```GerritJsonEventFactory``` and named something like ```getListOfStrings```.